services/galley/src/Galley/Data.hs

```diff
@@ -309,6 +310,13 @@ userTeams u =
   map runIdentity
     <$> retry x1 (query Cql.selectUserTeams (params Quorum (Identity u)))
 
+usersTeams :: MonadClient m => [UserId] -> m (Map UserId [TeamId])
```

Comment:
A user can only be member of one team:

```suggestion
usersTeams :: MonadClient m => [UserId] -> m (Map UserId TeamId)
```

(there are non-binding teams in the backend code, but they have never been supported by any client, never will be since there are now other plans that would work better if we ever needed such a thing, and really they should just be removed.)

Reply:
done